apps/molgenis-components/src/components/forms/InputOntology.vue

```diff
@@ -558,8 +560,8 @@ export default {
           description="please choose your options in tree below"
           v-model="value"
           :isMultiSelect="false"
-          tableName="Tag"
```

Comment:
has the demo been changes from Pet Store to a Catalogue?

Reply:
thx for spotting need to undo